src/Field.cpp

```diff
@@ -926,11 +931,7 @@ omnmmsgFieldPayload_getAsString       (const msgFieldPayload   field,
     {
     case MAMA_FIELD_TYPE_BOOL:
     {
-        EXPAND_PRINT_SCALAR_MACROS (mama_bool_t,
-                                    (mama_bool_t) 0,
-                                    Bool,
-                                    "%u",
-                                    mama_bool_t);
+        EXPAND_PRINT_SCALAR_MACROS (mama_bool_t, (mama_bool_t) 0, Bool, "%u", mama_bool_t);
```

Comment:
Please don't make unnecessary and unrelated formatting changes. I like the old way much better for this anyway.

Reply:
Gee Frank -- all I did was make them consistent.  

I assume by "old way" you mean split over several lines?  (There's about the same number of each style IIRC).

Reply:
For this area of code yes. These macros are unreadable enough as it is without squishing them.